lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -13,7 +13,7 @@
         <div class="cohort-management-group-setup">
             <div class="setup-value">
                 <% if (cohort.get('assignment_type') == "none") { %>
-                    <%- gettext("Students are added to this cohort group only when you provide their email addresses or usernames on this page.") %>
+                    <%- gettext("Students are added to this cohort group only when you provide their email addresses or username on this page.") %>
```

Comment:
I don't understand this change. Given we are using plural for "email addresses", we should be consistent with "usernames". Either email addresses or usernames can be provided.